Taskfile.yml

```diff
@@ -324,56 +322,50 @@ tasks:
       - pre-commit run -a
 
   run-website:
-    desc: Build & serve the static website.
+    desc: Build & serve the static website for interactive development.
     dir: web/website
     cmds:
       - hugo server
 
-  run-web:
-    desc: Build & serve the website & playground.
-    summary:
-      Note that this only live-reloads the static website; and requires
-      rerunning to pick up playground & book changes.
-    dir: web/website
-    cmds:
-      - task: build-web
-      # Then start web server with rendering to disk, so it picks up the playground files.
-      - hugo server --renderToDisk
-
   run-book:
-    desc: Build & serve the book.
+    desc: Build & serve the book for interactive development.
     dir: web/book
     cmds:
       - mdbook serve --port=3000
 
   run-playground:
-    desc: Build & serve the playground.
-    dir: web/playground
-    cmds:
-      - npm install
-      - npm start
-
-  run-playground-cached:
-    desc: Build & serve the playground, without rebuilding rust code.
+    desc: Build & serve the playground for interactive development.
     dir: web/playground
     cmds:
-      - task: install-playground-npm-dependencies
+      - task update-playground-dependencies
       - npm start
 
-  install-playground-npm-dependencies:
-    # Look to see if npm dependencies need to be updated
-    # Use task's sources/generates to see if checksums of
-    # node_modules directory have changed since package.json was updated
-
-    desc: Check to see if package.json has changed since we ran npm install
+  update-playground-dependencies:
+    # Check if npm dependencies for the playground need to be updated
+    # Use task's sources/generates to see if package.json,
+    # or anything in crates or bindings was updated after the
+    # node_modules was rebuilt
+    desc: Install dep's if something important changed recently
     dir: web/playground
     cmds:
-      - npm install
+      - task install-playground-dependencies-all
     sources:
       - package.json
+      - ../../crates/**/*.rs
+      - ../../bindings/**/*.rs
     generates:
       - node_modules/*
```

Comment:
Are you familiar with why we need this? Because it's actually generating much more than just this path — it's generating the built files for the playground. 

From the [Taskfile help](https://taskfile.dev/usage/#by-fingerprinting-locally-generated-files-and-their-sources) (not providing a conclusive answer):

> For the checksum (default) or timestamp method to work, it is only necessary to inform the source files. When the timestamp method is used, the last time of the running the task is considered as a generate.

We could try removing — often it's better to have nothing than have something partially correct...

Reply:
I think you're right - timestamps on sources seem to be sufficient. I'll remove it.